### ISSUE TITLE
Improper encoding characters in files

### ISSUE THREAD

**Issue by liuanna** [weecology/PortalData#21]
Hi folks,

My name is Anna Liu. I am working on some project which using the data from your repo. I realized that there are some improper encoding characters in multiple data files and README files. They are not encoded in UTF-8. I tried to read from github directly and also read from files after cloning to local using R. But I can still find improper encoding characters in files and they cause problems when I use them for my project.

For example, 
```
ant_species = getURL("https://raw.githubusercontent.com/weecology/PortalData/master/Ants/Portal_ant_species.csv")
```
returns

[1] "Species Code,Scientific Name,ID Issues\ncamp fest,Camponotus festinatus,\"Unclear if in other years this was simply labeled camp sp, or if camp sp is a different species\"\ncamp sp,Camponotus sp.,\ncono bico,Conomyrma bicolor,\ncono insa,Conomyrma insana,\ncrem sp,Crematogastor sp.,\nirid prui,Forelius pruinosus (Iridomyrma pruinosum),\"Prior to 1987, simply labeled Dolichoderinae, may include more than this species\"\nmyrm depi,Myrmecocystus depilis,\nmyrm mexi,Myrmecocystus mexicanus,\nmyrm mimi,Myrmecocystus mimicus,\"Prior to 1982, counts included all myrm mimi and myrm depi\"\nmyrm nava,Myrmecocystus navajo,\nnovo cock,Novomessor cockerelli,\nnovo sp,Novomessor spp.,\nphei dese,Pheidole desertorum,\nphei mili,Pheidole militicida,\nphei sp,Pheidole sp,\nphei sita,Pheidole sitarches,\nphei xero,Pheidole xerophila,\nphei yell,Pheidole sp **\x93yellow\x94**,No further description\npogo barb,Pogonomyrmex barbatus,\npogo dese,Pogonomyrmex desertorum,\npogo imbe,Pogonomyrmex imbebiculus,\npogo mari,Pogonomyrmex maricopa,\npogo pima,Pogonomyrmex pima,\npogo rugo,Pogonomyrmex rugosus,\nsole sp,Solenopsis sp.,\nsole xylo,Solenopsis xyloni,\nunkn,Unknown species,Possibly contains several species within one year and plot\n"

The bolded phrase **\x93yellow\x94** causes problem, since it is not encoded in UTF-8. In the original file in github repo, it looks like "yellow". I think, characters with improper encoding are contained inside the original files. 

Would you mind looking into these files and checking if some symbols and characters are not encoded in improper format? There are multiple files.

Thank you very much,

Anna

**Comment by ethanwhite** [comment]
Hi Anna - thanks for reporting this issue and helping us make the Portal data as clean as usable as possible.

Looks like there were some fancy quotes from a word processor from years ago. I just opened a PR (#22) to replace those with standard quotation marks. One of the Portal folks will check this and get the PR merged if it looks good and you should be all set.

**Comment by skmorgane** [comment]
I've merged in Ethan's changes from PR (#22). Hopefully, the data will import cleanly for you now!

**Comment by liuanna** [comment]
Thank you!
